Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -185,6 +190,7 @@ public class NetworkManager : MonoBehaviour
         /// <summary>
         /// headless mode detection
         /// </summary>
+        [System.Obsolete("Use #if UNITY_SERVER instead.")]
```

Comment:
for cleanliness, could we use [Obsolete] instead of [System.Obsolete] please? we do this in the other places too :)